utils/swift_build_support/swift_build_support/products/tensorflow.py

```diff
@@ -62,31 +62,48 @@ def build(self, host_target):
 
         with shell.pushd(self.build_dir):
 
-            shell.call([
-                self.toolchain.cmake,
-                '-G', 'Ninja',
-                '-D', 'BUILD_SHARED_LIBS=YES',
-                '-D', 'CMAKE_INSTALL_PREFIX={}/usr'.format(
-                    self.args.install_destdir),
-                '-D', 'CMAKE_MAKE_PROGRAM={}'.format(self.toolchain.ninja),
-                '-D', 'CMAKE_Swift_COMPILER={}'.format(swiftc),
-                # SWIFT_ENABLE_TENSORFLOW
-                '-D', 'USE_BUNDLED_CTENSORFLOW=YES',
-                # SWIFT_ENABLE_TENSORFLOW END
-                '-D', 'TensorFlow_INCLUDE_DIR={}'.format(tensorflow_source_dir),
-                '-D', 'TensorFlow_LIBRARY={}'.format(
-                    os.path.join(tensorflow_source_dir, 'bazel-bin', 'tensorflow',
-                                 lib_name)),
-                '-D', 'CMAKE_Swift_FLAGS={}'.format('-L{}'.format(
-                    os.path.join(tensorflow_source_dir, 'bazel-bin', 'tensorflow'))
-                ),
-                '-B', self.build_dir,
-                '-S', self.source_dir,
-            ])
-            shell.call([
-                self.toolchain.cmake,
-                '--build', self.build_dir,
-            ])
+            shell.call(
```

Comment:
Could you have black not reformat this?  This makes the command much more difficult to understand

Reply:
Done. I reverted `black` changes.